jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Convert categorical columns to strings
</pre><p>can't you just define a list of the things you want to transform and then loop through it? Would greatly reduce the amount of code here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I guess that I can. Okay, I'll change it to a for loop.</p>